theano/tensor/nnet/abstract_conv.py

```diff
@@ -422,7 +422,7 @@ def conv2d_grad_wrt_weights(input,
                                                filter_flip=filter_flip,
                                                filter_dilation=filter_dilation)
 
-    return gradWeight_op(input, output_grad, filter_shape[:-2])
```

Comment:
It needs a new test, but the test will be introduced with #4711  pull request.

It fixes the issue #4885